cm-server/src/main/java/ro/msg/cm/controller/ImportDatabase.java

```diff
@@ -0,0 +1,111 @@
+        importCSV(Tag.class, request.getInputStream(), this.tagRepository);
+    }
+
+    private void printRequestDetail(boolean debug, HttpServletRequest request) throws IOException {
+        if (!debug) return;
+        System.out.print(request.getMethod());
+        Enumeration params = request.getParameterNames();
+        while (params.hasMoreElements()) {
+            String paramName = (String) params.nextElement();
+            System.out.println("Parameter Name - " + paramName + ", Value - " + request.getParameter(paramName));
+        }
+        Enumeration headerNames = request.getHeaderNames();
+        while (headerNames.hasMoreElements()) {
+            String headerName = (String) headerNames.nextElement();
+            System.out.println("Header Name - " + headerName + ", Value - " + request.getHeader(headerName));
+        }
+        String result = new BufferedReader(new InputStreamReader(request.getInputStream()))
+                .lines().collect(Collectors.joining("\n"));
+        System.out.println(result.length());
+        System.out.println(result);
+
+    }
+
+    private void importCSV(Class table, InputStream csvContent, CrudRepository rep) throws IOException {
+        String line ;
+        String csvSplitBy = ",";
+        String[] headers = null;
+        BufferedReader in = new BufferedReader(new InputStreamReader(csvContent));
+        try {
+            while ((line = in.readLine()) != null) {
+                if (headers == null) {
+                    headers = line.split(csvSplitBy);
+                    System.out.println(Arrays.toString(headers));
+                    continue;
+                }
+                String[] elements = line.split(csvSplitBy);
+                Object importObject = table.newInstance();
+                Method[] mts = table.getDeclaredMethods();
+                for (Method m : mts) {
+                    for (int index = 0; index < headers.length; index++) {
+                        String headerElement = headers[index].replaceAll("_","").toLowerCase();
+                        if ((m.getName().toLowerCase()).startsWith("set" + headerElement)) {
+                            try {
+                                Class<?>[] parameterTypes = m.getParameterTypes();
+                                if (parameterTypes[0].isPrimitive()) {
+                                    m.invoke(importObject, Integer.parseInt(elements[index]));
+                                } else {
+                                    m.invoke(importObject, elements[index]);
+                                }
+                                // Handle any exceptions thrown by method to be invoked.
+                            } catch (Exception x) {
+                                x.printStackTrace();
+                            }
+                        }
+                    }
+                }
+                rep.save(importObject);
+            }
+
+        } catch (IOException e) {
```

Comment:
Log exceptions instead of printStackTrace. Let's use [Lombok's SLF4J annotation](https://projectlombok.org/features/log) to create the logger for us.